gazebo_plugins/test/worlds/gazebo_ros_gps_sensor.world

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+<sdf version="1.6">
+  <world name="gazebo_ros_gps_sensor_world">
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <spherical_coordinates>
+        <!-- currently gazebo has a bug: instead of outputing lat, long, altitude in ENU as the
+        default configurations, it's outputting (-E)(-N)U, therefore we rotate the default frame
+        180 so that it would go back to 180 -->
```

Comment:
Mind adding a link to this issue here so we don't lose track of it?

https://bitbucket.org/osrf/gazebo/issues/2022/

Reply:
I didn't search about it before. Good to know there is already an issue on this!

Reply:
Sorry @chapulina, at the time I was making the changes I didn't understand that you wanted me to add this link https://bitbucket.org/osrf/gazebo/issues/2022/ on the .world comment. Now that it has been merged into master, maybe you can add it directly.

Reply:
No worries, it can be added later :wink: